doc/MANUAL.md

```diff
@@ -128,29 +160,41 @@ If this is not possible for some reason, search for the CVE number to get
 information on possible workarounds and patches that could be backported to
 other versions.  Note that neither workarounds nor backported fixes can be
 detected by this tool, so your binary will continue to show up as vulnerable
-even though it may now be safely mitigated and the result a false positive.
+even though it may now be safely mitigated and result in a false positive.
+To avoid this problem, we recommend classifing CVE as Mitigated as explained
+in Input section.
 
-Limitations
------------
+## Limitations
 
-The last release of this tool to support python 2.7 is 0.3.1.  Please use
-python 3.6+ for development and future versions.   Linux and Windows are
+The last release of this tool to support python 2.7 is 0.3.1. Please use
```

Comment:
Again, I think it's a waste of time to switch to one space after period when most of the doc is two and the rendering will look the same anyhow.